CMakeLists.txt

```diff
@@ -233,6 +234,11 @@ else()
 				set(CMAKE_C_FLAGS "-O0 -g")
 			endif()
 		endif()
+		if(WANT_SANITIZER)
```

Comment:
This is specifically address sanitizer. There are other sanitizers - TSan, UBSan, MSan, etc.

I don't mind being able to enable ASan using a special option, though I usually set CXXFLAGS directly in such cases. But I suggest at least making it clear somehow which sanitizer is used: either `WANT_ADDRESS_SANITIZER=1` or `WANT_SANITIZER=address`

Reply:
Nice idea for improvement, will implement

Reply:
Done in https://github.com/kvirc/KVIrc/pull/2599/commits/962c93a66e15bc8748e77a4b53d6f06cb4c725e8
Btw gcc unfortunately doesn't support the memory sanitizer, only clang does.